docs/application/dotnet/guides/nui/visuals.md

```diff
@@ -1,390 +1,552 @@
-| `Units`         | `integer` or `string` | No              | The coordinate system.                   |
-| `SpreadMethod`  | `integer` or `string` | No              | Indicates what happens if a gradient starts or ends inside bounds. |
-
-### Units
-The `Units` are used to define the coordinate system for the attributes:
- -  Start (x1, y1) and end (x2, y2) points of a line, if using a linear gradient.
- -  Center point (cx, cy) and radius (r) of a circle, if using a radial gradient.
-
-### SpreadMethod
-The `SpreadMethod` property indicates what happens if the gradient starts or ends inside the bounds of the target rectangle.
-
-**Table: SpreadMethod values**
-
-| Enumeration value | Description                              |
-| ----------------- | ---------------------------------------- |
-| `Pad`             | Default, uses the terminal colors of the gradient to fill the remainder of the quad. |
-| `Reflect`         | Reflects the gradient pattern start-to-end, end-to-start, start-to-end, and so on, until the quad is filled. |
-| `Repeat`          | Repeats the gradient pattern start-to-end, start-to-end, start-to-end, and so on, until the quad is filled. |
-
-<a name="gradientusage"></a>
+| Property        | Type                             | Required        | Description                                                                                 |
+|-----------------|----------------------------------|-----------------|---------------------------------------------------------------------------------------------|
+| `StartPosition` | `Vector2`                        | For linear only | The start position of the linear gradient - coordinate system depends on the `Units` value. |
+| `EndPosition`   | `Vector2`                        | For linear only | The end position of the linear gradient - coordinate system depends on the `Units` value.   |
+| `Center`        | `Vector2`                        | For radial only | The center point of the radial gradient - coordinate system depends on the `Units` value.   |
+| `Radius`        | `float`                          | For radial only | The size of the radial gradient radius - coordinate system depends on the `Units` value.    |
+| `StopColor`     | `PropertyArray` of `Color`       | Yes             | The color at the stop offsets. At least two are required to show a gradient.                |
+| `StopOffset`    | `PropertyArray` of `floats`      | No              | The stop offsets in relative units (default is 0.0 and 1.0) - to see all colors from the `StopColor` the lengths of these two arrays should be the same.            |
+| `Units`         | `GradientVisualUnitsType`        | No              | Defines the coordinate system for the attributes: start and end points for a linear gradient, center point and radius for a radial gradient (see tab. [with possible values](#table_grad_possible_values)). |
```

Comment:
```suggestion
| `Units`         | `GradientVisualUnitsType`        | No              | Defines the coordinate system for the attributes: start and end points for a linear gradient, center point and radius for a radial gradient (For more information, see [possible values](#table_grad_possible_values). |

Reply:
please check now:

Defines the coordinate system for the attributes: start and end points for a linear gradient, center point and radius for a radial gradient (default value - `ObjectBoundingBox`; see the [below table](#table_grad_possible_values_units) for more details).